tests/test_ml_tools.py

```diff
@@ -0,0 +1,217 @@
+import awkward as ak
+import dask_awkward as dak
+import numpy as np
+import pytest
+from distributed import Client
+
+
+def prepare_jets_array(njets):
+    # Creating jagged Jet-with-constituent array, returning both awkward and lazy
+    # dask_awkward arrays
+    NFEAT = 100
+    jets = ak.zip(
+        {
+            "pt": ak.from_numpy(np.random.random(size=njets)),
+            "eta": ak.from_numpy(np.random.random(size=njets)),
+            "phi": ak.from_numpy(np.random.random(size=njets)),
+            "ncands": ak.from_numpy(np.random.randint(1, 50, size=njets)),
+        },
+        with_name="LorentzVector",
```

Comment:
The `with_name` here doesn't do anything because you don't also set `behavior=coffea.nanoevents.methods.vector.behavior`. Which is good because `LorentzVector` mixin expects `x,y,z,t` attributes and this wouldn't work without using `PtEtaPhiMLorentzVector` as the mixin.

Reply:
Removing, as Lorentz-Vector structure is not strictly needed for testing.